### DISCUSSION TITLE
Seattle Hospital Location Finder

### DISCUSSION THREAD

**Discussion by whalen323** [jakobzhao/geog495#75] [Final Project]
Super late I know, but I'm just happy to have finished!

[Application link](https://whalen323.github.io/finalgeog495-main/) 
[Github link](https://github.com/whalen323/finalgeog495-main )

My web application is a location finder for Seattle area hospitals. It displays a scrollable list of all the hospitals on the left-hand side panel, with the map on the right. You can click on a hospital name to zoom into its location and see its street address. I also added a location searching tool on the upper right-hand side. You can type a street or building name into the box and select search and the application will take you to that location, display whatever the closest hospital is to what you searched in the window, and then sort the remainder of the list based on distance from your point. 

Screenshot: 
![startup](https://user-images.githubusercontent.com/92193901/145749441-1e125f2d-5e08-45a5-b4c7-d64804c63515.png)

**Comment by Conibaer** [comment]
The nearest hospital is obviously great information to know. Some of the design choices might be fighting each other on this map though. Matching the icon red to the left hand panel red would be a great start. Adding an exit ability from your popups could also come in handy

**Comment by Stephenwzy** [comment]
I like the red coloring style for both markers and the listing items. This application can convey information! Maybe you can make the markers' background transparent since I notice that there are two little white corners on markers. Overall, this is definitely an application with a high degree of completeness!

**Comment by madiframpton** [comment]
Hi! Great work on your map. I love the color scheme and your base-map looks super cool! I also thought it was a great idea to add how far away each hospital is in the sidebar. Really nicely done!

**Comment by Anemnox** [comment]
I really like the custom markers for the hospitals! They're easily recognizable as hospitals and fit the theme of the map. It's really cool to see that you've implemented a search function for the hospitals too! One thing that I would say is that it would be nice to have an exit function for the icon popups but overall the map looks fantastic!

**Comment by Zhiyuan-Jiao** [comment]
Hello. I really like how you use those little nice icon to represent the hospital. I think the map will be quite helpful for those searching for a hospital near them. It will be even better if you could add a sorting by distance function so that the user could find the closest hospital easily.
